src/index.html

```diff
@@ -8,6 +8,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
+
+    <div class="frame">
+      <div class="container">
+        <div class="stars--0">
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+        </div>
+        <div class="stars--1">
+          <div class="stars_star--active"></div>
+
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
```

Comment:
same here, at the first u shoud have `stars_star` and only than modifyer `stars_star--active`